src/lib/constants.js

```diff
@@ -11,9 +13,15 @@ const WEEKEND = [
   /week\-?end/i,
 ]
 
+
+// other patterns
+const USER_ALREADY_INTRODUCED = /suis|am|\'m/i
```

Comment:
I don't know if it's possible but I think the best approach here, is when a user joins the team, he receives a Direct message from the bot saying something like : `Hello, je suis le bot gardien de cette communauté, voici quelques règles à savoir sur la communauté <drop link guidelines et quelques explications sur les channels>, mais avant tout, présente toi sur #chkoun`

Is this possible, I think it will be awesome if we can achieve this